### PR TITLE
Fix 1095374: Resource-level fencing using CIB

### PR DESCRIPTION
**Pull request by tomschr**
This PR contains the following changes:

* Add new section (ID=sec.ha.drbd.fencing) with example of `net` and `handlers` section in DRBD config file

### CODE REVIEW

File: xml/ha_drbd.xml [hunk 1]
```diff
@@ -927,6 +927,57 @@ resource r0-U {
...
+ <sect1 xml:id="sec.ha.drbd.fencing">
+  <title>Using Resource-Level Fencing</title>
+  <para>
+   When a DRBD replication link becomes interrupted, &pace; tries to promote
+   the DRBD resource to another node. To prevent &pace; from starting a service
+   with outdated data, enable resource-level fencing in the DRBD configuration
+   file as shown in <xref linkend="ex.ha.drbd.fencing"/>.
+  </para>
+  <example xml:id="ex.ha.drbd.fencing">
+   <title>Configuration of DRBD with Resource-Level Fencing using the Cluster
```
Comment (taroth21) [inline xml/ha_drbd.xml]: I think in title style spelling, 'using' should be written 'Using'

File: xml/ha_drbd.xml [hunk 2]
```diff
@@ -927,6 +927,57 @@ resource r0-U {
...
+   <listitem>
+    <para>The script contacts the cluster manager.
+    </para>
+   </listitem>
+   <listitem>
+    <para>The script determines the &pace; resource associated with this
+     DRBD resource.</para>
+   </listitem>
+   <listitem>
+    <para>The script ensures, that the DRBD resource no longer gets
```
Comment (taroth21) [inline xml/ha_drbd.xml]: no comma

File: xml/ha_drbd.xml [hunk 3]
```diff
@@ -927,6 +927,57 @@ resource r0-U {
...
+    promoted to any other node. It stays on the currently active one.</para>
+   </listitem>
+   <listitem>
+    <para>If the replication link becomes connected again and DRBD
+    completes its synchronization process, then the contraint is removed.
```
Comment (taroth21) [inline xml/ha_drbd.xml]: contraint - > constraint

**Comment by tomschr** [comment]
@taroth21 Thanks for the review! Could you approve my changes, so everything get green? :wink: Thanks!

**Comment by tomschr** [comment]
Reopen the branch until Olli in bsc#1095374 is finished.

**Pull request merged**